kiota_abstractions/serialization/untyped_list.py

```diff
@@ -0,0 +1,24 @@
+from .untyped_node import UntypedNode
+
+class UntypedList(UntypedNode):
+    """
+    Represents an untyped node with list value.
+    """
+
+    def __init__(self, value: list) -> None:
+        """
+        Creates a new instance of UntypedList.
+
+        Args:
+            value (bool): The list value associated with the node.
+        """
+        self.__value = value
+
+    def get_value(self) -> list:
```

Comment:
Is it possible to constraing this to a list of `UntypedNode`?

Reply:
Done